### PR TITLE
NXP-25625: Add docker compose file for postgresql 11

### PR DESCRIPTION
**Pull request by kevinleturc**

### CODE REVIEW

**Comment by kevinleturc** [comment]
Builds below have been successful or has same result than postgresql builds for version 9.6:
- https://qa.nuxeo.org/jenkins/job/master/job/nuxeo-master-fullbuild-part1-pgsql-11/1/
- https://qa.nuxeo.org/jenkins/job/master/job/FT-nuxeo-master-server-tomcat-pgsql-11/1/

File: integration/Jenkinsfiles/docker-compose-pgsql-11.yml [hunk 1]
```diff
@@ -0,0 +1,38 @@
+#
+# (C) Copyright 2017 Nuxeo (http://nuxeo.com/) and others.
```
Comment (funshodavid) [inline integration/Jenkinsfiles/docker-compose-pgsql-11.yml]: Wrong year

**Comment by bdelbosc** [comment]
:+1:

**Comment by funshodavid** [comment]
:+1:

**Pull request merged**